charts/core-dump-handler/README.md

```diff
@@ -228,3 +228,38 @@ Daemonset
 * s3BucketName : Maps to the S3_BUCKET_NAME enviroment variable
 * 3Region : Maps to the S3_REGION enviroment variable
 * extraEnvVars: Option for passing additional configuration to the agent such as endpoint properties.
+
+### EKS setup for gitops pipelines (`eksctl` or similar)
```

Comment:
Can you move this section above the environment section
https://github.com/IBM/core-dump-handler/pull/76/files/5cf0b0b02e9cce21502aa594536cb951f6c5a5ef#diff-35330df803a4bbfce6b8b734490d5f0b77bb117755aeac1958500a4934a86bb8R104